src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -1705,7 +1705,7 @@ private static Location GetInterfaceLocation(Symbol interfaceMember, TypeSymbol
                 snt = implementingType as SourceMemberContainerTypeSymbol;
             }
 
-            return snt?.GetImplementsLocation(@interface) ?? implementingType.Locations[0];
+            return snt?.GetImplementsLocation(@interface) ?? implementingType.Locations.FirstOrDefault();
```

Comment:
>FirstOrDefault [](start = 88, length = 14)

potentially returning `null` Location here seems risky. Let's return `Location.None` instead in the worst case